README.md

```diff
@@ -7,6 +7,7 @@ Cool Hackathons for college students
 - ([Apply Now](https://blog.ycombinator.com/apply-to-the-yc-fall-hackathon-2019/)) **Y Combinator Hackathon, November 22, 2019, YC
 Mountain View, CA**
 - ([Apply Now](https://www.mitrealityhack.com/)) **MIT Reality Hack, January 16-20, 2020, MIT, Cambridge, MA**
+- ([Apply Now](https://www.register.tamuhack.com/)) **TAMUhack, January 25-26, 2020, Texas A&M University, College Station, TX**
```

Comment:
```suggestion
- ([Apply Now](https://register.tamuhack.com/)) **TAMUhack, January 25-26, 2020, Texas A&M University, College Station, TX**
```

Reply:
<img width="1093" alt="Screen Shot 2019-11-29 at 3 07 10 PM" src="https://user-images.githubusercontent.com/3537801/69892403-f365b480-12b9-11ea-8df4-1995380715f9.png">

It seems that the link shouldn't be  prefixed with `www`